runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsService.java

```diff
@@ -30,13 +30,37 @@
 import javax.inject.Singleton;
 import net.runelite.api.Client;
 import net.runelite.api.Player;
+import net.runelite.client.util.ColorUtil;
+
+import java.util.concurrent.ConcurrentHashMap;
 
 @Singleton
 public class PlayerIndicatorsService
 {
 	private final Client client;
 	private final PlayerIndicatorsConfig config;
 
+	private final int MAX_SIZE = 2500;
+	private final ConcurrentHashMap<String, Color> colorMap = new ConcurrentHashMap<>();
```

Comment:
A `LoadingCache` seems like a better structure to be used here, as you can set a `maximumSize` on that, rather than manually removing elements once a map reaches max size.

Reply:
This do not needs to use cache at all anymore since its now coloring based on name hash, so this should just be removed

Reply:
Alright then, will remove it, checked the framerate, didn't seem to budge much with / without it, so cheers